### DISCUSSION TITLE
Vs Trollge

### DISCUSSION THREAD

**Discussion by Kaique62** [luckydog7/Funkin-android#1490] [mods]
Link: https://youtu.be/E955g55Q9t0
Screenshots:
![Screenshot_20210628-224344](https://user-images.githubusercontent.com/81991014/123724322-85e98380-d862-11eb-97ca-5e01f82fbe44.png)
![Screenshot_20210628-224406](https://user-images.githubusercontent.com/81991014/123724335-8c77fb00-d862-11eb-89f6-b72cd9e946ec.png)

**Comment by DzGithub2805** [comment]
Thx, i always wanted this :trollface:

**Comment by Kappandrew** [comment]
We do a little moddin' :trollface:
